### ISSUE TITLE
Where/what is scalac?

### ISSUE THREAD

**Issue by qwertystop** [w0rp/ale#1302]
The default fixers for Scala are scalac and scalastyle. I've found scalastyle, but not scalac - all searches for such a thing refer to either a software development company by that name, or instructions that at some point involve using the `scalac` command but assume you have it. I followed the instructions on the scala-lang website to install `sbt`, and I have no `scalac` binary on my PATH. I can't find instructions for getting it, either.

**Comment by elebow** [comment]
The sbt documentation at https://github.com/scala/scala#using-the-sbt-build makes reference to `scalac` as a "core command", so maybe it's just not being added to your PATH?

**Comment by qwertystop** [comment]
That's a command in sbt, not something you can call straight from the command line.

I installed `scala` from apt after finding that `scala` and `scalac` are basically the equivalent to Java's similarly-named commands; that gave me `scala` and `scalac` binaries, but now I'm basically managing two parallel installations which is suboptimal. And it turns out that `scalac` integration with ALE doesn't even catch most of the errors that sbt's compilation does, so I'm not getting much benefit from it, but I see there's already an issue for that (#1283).

In short: Resolved as not-an-ALE-issue, it's just Scala's devs making two separate ways to install/compile/run Scala programs for some reason, and ALE only integrates with one of them which happened not to be the one that's currently recommended to new users.
